dataset/src/test/scala/frameless/ColumnTests.scala

```diff
@@ -45,6 +45,22 @@ class ColumnTests extends TypedDatasetSuite {
     check(forAll(prop[SQLTimestamp] _))
     check(forAll(prop[String] _))
     check(forAll(prop[Instant] _))
+
+      /*
+      Fails at runtime!!
```

Comment:
This would currently fail in spark `2.2` though testing on spark `2.3-SNAPSHOT` seems to pass. Not sure how to handle this in the meantime.